### ISSUE TITLE
Hash Key Can't Start With Uppercase Bug

### ISSUE THREAD

**Issue by Maxwell-Alexius** [goby-lang/goby#343]
Error log:
```
Goby 0.0.9 😽 😝 😄
» { taipei: 101 }
#» { taipei: 101 }
» { Taipei: 101 }
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x43ec7b9]

goroutine 1 [running]:
github.com/goby-lang/goby/igb.StartIgb(0x44fb66b, 0x5)
        /Users/MaxwellAlexius/go/src/github.com/goby-lang/goby/igb/repl.go:204 +0x1029
main.main()
        /Users/MaxwellAlexius/go/src/github.com/goby-lang/goby/goby.go:28 +0x743
exit status 2
```

**Comment by st0012** [comment]
@Maxwell-Alexius This should be fixed. Please help me confirm it.

**Comment by Maxwell-Alexius** [comment]
Closed with confirm fixed in commit 83f74aed3c671a8c47fccb1fd717b276f8004d60
